src/redux/StudentEdit.redux/StudentEdit.action.js

```diff
@@ -0,0 +1,17 @@
+import Axios from 'axios';
+
+export const EDIT_USER_START = 'EDIT_USER_START';
```

Comment:
let's move these over to a file named `StudentEdit.types.js`

Reply:
also, this file should named `StudentEdit.actions.js`, plural

Reply:
done